lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -314,4 +315,33 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
         });
     return Container();
   }
+
+  void settingModalBottomSheet(BuildContext context, Document document) {
+    showModalBottomSheet(
```

Comment:
I did not find the file info in the header.

Reply:
One more things, we will have the context menu in several places. I don't want to duplicate the code. How about a builder for this kind of widget?

Reply:
Do you have an idea to bind function to differents tiles ? ex: My Space context menu Download will call my space view model etc

Reply:
Found a clean solution tell me what you think https://github.com/linagora/linshare-mobile-flutter-app/pull/13/commits/760c6e9470df3d2c658cd46bfba57b3970c67d8f